public/css/app.css

```diff
@@ -56,6 +56,9 @@ body, html {
     background: #05497B;
     color:white;
     font-family: Arial, Helvetica, sans-serif;
+    position: absolute;
```

Comment:
If this is absolute, then it doesn't work with smaller screen sizes (the footer will be in the middle of the content). 
![image](https://user-images.githubusercontent.com/52092026/159170409-eb55071d-703c-4f9c-ae5b-bc96844fe7e4.png)
(Sorry I didn't catch this problem on last review)
Either leave it as relative (which is uglier on bigger screens) or find another way to make it at the bottom only.